main/src/com/google/refine/importers/XmlImporter.java

```diff
@@ -216,7 +219,45 @@ static public class XmlParser implements TreeReader {
         static final int WHITESPACE_CHARACTERS_TOKEN = 15;
 
         public XmlParser(InputStream inputStream) throws XMLStreamException, IOException {
-            parser = createXMLStreamReader(inputStream);
+            parser = createXMLStreamReader(new InvalidXmlFilterInputStream(inputStream));
+        }
+
+        static class InvalidXmlFilterInputStream extends BufferedInputStream {
+
+            private final Pattern pattern = Pattern.compile("[^\\u0009\\u000A\\u000D\\u0020-\\uD7FF\\uE000-\\uFFFD\\u10000-\\u10FFF]+");
```

Comment:
Many of these characters are legal in the CDATA elements of XML 1.1 documents https://www.w3.org/TR/xml11/#NT-Char